src/scales/scale.linearbase.js

```diff
@@ -6,44 +6,65 @@ module.exports = function(Chart) {
 		noop = helpers.noop;
 
 	Chart.LinearScaleBase = Chart.Scale.extend({
+		// Following http://www.erichynds.com/blog/backbone-and-inheritance
+		constructor: function() {
+			// These two properties kept for backwards compatibility
+			var me = this;
+			Object.defineProperty(me, 'min', {
```

Comment:
mostly so that I was able to only pass the `dataRange` to the generator rather than passing the entire scale object